docs/build/guides/contract-accounts/smart-wallets/overview.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Smart wallets
```

Comment:
Smart wallets should be renamed to Contract accounts to be more consistent. Please consider doing this more globally, thank you.